### ISSUE TITLE
tensorflow-gpu 2.3.1 not available

### ISSUE THREAD

**Issue by BlueskyFR** [ContinuumIO/anaconda-issues#12163]
Why isn't **tensorflow-gpu 2.3.1** not yet available in conda?

**Comment by csoja** [comment]
My understand is that our current Linux compilers do not support building tensorflow-gpu 2.3.1.  We need to update our compilers first, and then we will be able to build tensorflow-gpu.  We do plan to do so, but have not gotten to it yet.

**Comment by BlueskyFR** [comment]
I saw that Tensorflow-gpu is not required anymore as it is now part of the base tensorflow package isn't it?

**Comment by katietz** [comment]
Yes, right now we don't support GPU variant for Linux.  Issue is that we fail to build gpu-variant with current toolchains we support.
First target supporting gpu for 2.3 will be Windows (just building packages for it).  @BlueskyFR  where you got this information from? We don't build with gpu variant right now.  Only mkl and eigen variants are present for Linux

**Comment by BlueskyFR** [comment]
> Yes, right now we don't support GPU variant for Linux.  Issue is that we fail to build gpu-variant with current toolchains we support.
> First target supporting gpu for 2.3 will be Windows (just building packages for it).  @BlueskyFR  where you got this information from? We don't build with gpu variant right now.  Only mkl and eigen variants are present for Linux
> 

The official Tensorflow website says that the recent releases of Tensorflow now includes gpu support out-of-the-box.
